container-images/tcib/base/os/watcher-base/watcher-base.yaml

```diff
@@ -0,0 +1,10 @@
+tcib_actions:
+- run: bash /usr/local/bin/uid_gid_manage watcher
+- run: dnf -y install {{ tcib_packages['common'] | join(' ') }} && dnf clean all && rm -rf /var/cache/dnf
+- run: sed -i -r 's,^(Listen 80),#\1,' /etc/httpd/conf/httpd.conf && sed -i -r 's,^(Listen 443),#\1,' /etc/httpd/conf.d/ssl.conf
+tcib_packages:
+  common:
+  - httpd
+  - mod_ssl
+  - openstack-watcher-common
+  - python3-mod_wsgi
```

Comment:
Only the watcher-api runs as a wsgi in httpd, I'd move httpd, mod_ssl and python3-mod_wsgi to the watcher-api container.

Reply:
Thank you for the updating in the pr!